terraform/stacks/umccr_data_portal/status_page.tf

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+#
+# Note: data-portal-status-page  app reuse some Portal TF created resources such as Cognito. 
+# This streamlines integrate UI login to the same authority.
+
+locals {
+
+  app_name   = "data-portal-status-page"
+  sub_domain = "status.data"
+
+  data_portal_status_page_app_domain = "${local.sub_domain}.${var.base_domain[terraform.workspace]}"
+
+  data_portal_status_page_alias_domain = {
+    prod = "status.data.umccr.org"
+    dev  = ""
+  }
+
+  data_portal_status_page_callback_urls = {
+    prod = ["https://${local.data_portal_status_page_app_domain}", "https://${local.data_portal_status_page_alias_domain[terraform.workspace]}"]
+    dev  = ["https://${local.data_portal_status_page_app_domain}"]
+  }
+
+  data_portal_status_page_oauth_redirect_url = {
+    prod = "https://${local.data_portal_status_page_alias_domain[terraform.workspace]}"
+    dev  = "https://${local.data_portal_status_page_app_domain}"
+  }
+
+  ssm_parameter_prefix = "/data_portal_status_page"
```

Comment:
Let break this down to data portal namespace for now. i.e pls change it to `/data_portal/status_page`